sql/wsrep_high_priority_service.cc

```diff
@@ -495,11 +497,11 @@ int Wsrep_applier_service::apply_write_set(const wsrep::ws_meta& ws_meta,
   DBUG_RETURN(ret);
 }
 
-void Wsrep_applier_service::after_apply()
+int Wsrep_applier_service::after_apply(const wsrep::const_buffer&)
```

Comment:
Why you change this to int when you always return 0 ?

Reply:
No, not always. `after_apply()` is actually calling methods that can fail (even without voting support). Thus having `after_apply()` to return `void` is a bug even now. It should return `int`.

Reply:
But indeed, fixed to return the actual error code.